src/style.css

```diff
@@ -0,0 +1,3 @@
+@import url(./styles/reset.css);
```

Comment:
Add quotes in all places
```suggestion
@import url('./styles/reset.css');
```